service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt

```diff
@@ -467,6 +467,7 @@ class AttendanceReservationController(
         val groupId: GroupId?,
         val absent: Boolean,
         val outOnBackupPlacement: Boolean,
+        val inBackupPlacement: Boolean,
```

Comment:
`outOnBackupPlacement` and `inBackupPlacement` booleans allow 4 different state combinations, but only 3 of them are sensible. Would enum be better?

Reply:
Done.

Reply:
👍